recipes/jill/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "jill" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jill-{{ version }}.tar.gz
+    sha256: 48ea86a508f87dab0293d00634b4e8c6677cb00676e617b6c46345bc10af5e10
+  #- url: https://raw.githubusercontent.com/johnnychen94/jill.py/master/requirements.txt
```

Comment:
Why is this needed? The file should be included in the package. If not, the source should probably be changed to GitHub and upstream notified.

Reply:
Yes, that will be the best solution: fixing it at the source. But before I do that, I wanted to see if that's the only thing missing. Left this commented as the source is being pulled from PyPI, pulling this file from the upstream-repo could make it pull a different version of the file. But, leaving it as a comment there helps as a reminder to raise an issue with upstream.

Reply:
Ahh ok. Then I would just raise the issue now and remove this line as it's cleaner that way. Conda-forge recipes should be as clean as possible as it sometimes can confuse the bots or introduce other overlooked points of failure – just as a best practice.